src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -115,8 +113,6 @@ case class AppDefinition(
 
   val portNumbers: Seq[Int] = portDefinitions.map(_.port)
 
-  override val isResident: Boolean = residency.isDefined
```

Comment:
🤨 don't we still have resident instances?

Reply:
well, now we have this at the RunSpec level.